proto/service.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package proto;
```

Comment:
This package name doesn't fit, it should be something like `files`.

Reply:
There's no connection between the folder that this file's inside of and the name of the package, this isn't Go.

Reply:
And I think the file name should be `files.proto`, not `service.proto`, but that's less important since there aren't anymore proto files in this server.